Doc/library/functions.rst

```diff
@@ -1251,7 +1251,7 @@ are always available.  They are listed here in alphabetical order.
    If *closefd* is ``False`` and a file descriptor rather than a filename was
    given, the underlying file descriptor will be kept open when the file is
    closed.  If a filename is given *closefd* must be ``True`` (the default)
-   otherwise an error will be raised.
+   otherwise, an error will be raised.
```

Comment:
There seems to be a separator (comma or semicolon) missing before "otherwise".

Reply:
Fixed!